migration-helpers/update-api-folder-structure.js

```diff
@@ -0,0 +1,254 @@
+/**
+ * Migrate API folder structure to v4
+ */
+
+const { resolve, join, basename } = require("path");
+const fs = require("fs-extra");
+const _ = require("lodash");
+_.mixin(require("lodash-inflection"));
+const { inspect } = require("util");
+const runJsCodeshift = require("../utils/runJsCodeshift");
+
+const normalizeName = _.kebabCase;
+
+/**
+ * @description Migrates settings.json to schema.json
+ *
+ * @param {string} apiPath Path to the current api
+ * @param {string} contentTypeName Name of the current contentType
+ */
+const convertModelToContentType = async (apiPath, contentTypeName) => {
+  const settingsJsonPath = join(
+    apiPath,
+    "models",
+    `${contentTypeName}.settings.json`
+  );
+
+  const exists = await fs.exists(settingsJsonPath);
+  if (!exists) {
+    console.error(`error: ${contentTypeName}.settings.json does not exist`);
+    return;
+  }
+
+  const v4SchemaJsonPath = join(
+    apiPath,
+    "content-types",
+    contentTypeName,
+    "schema.json"
+  );
+
+  try {
+    // Read the settings.json file
+    const settingsJson = await fs.readJson(settingsJsonPath);
+    // Create a copy
+    const schemaJson = { ...settingsJson };
+    const infoUpdate = {
+      singularName: contentTypeName,
```

Comment:
We also need to make sure that `contentTypeName` is indeed singular. You can use `pluralize.isPlural(contentTypeName)` for this, and if it is, then `pluralize.singular(contentTypeName)`

Reply:
Couldn't I just call `pluralize.singular(contentTypeName)` ?  I don't think the check is necessary, if it's already singular it will remain singular.  What do you think?